opendatahub-io/data-science-pipelines-operator
# Set MariaDB and Envoy Proxy with their sha digests

## The issue resolved by this Pull Request:
Resolves [RHOAIENG-3565](https://issues.redhat.com/browse/RHOAIENG-3565)

## Description of your changes:
Change MariaDB and Envoy Proxy to point to the image sha digests.

## Testing instructions
* Deploy DSPO with the image referenced in this PR
* Create a DSPA image without setting mariadb and envoy proxy images
* Check if Deployment objects for both are pointing to their sha digests

## Checklist
